libraries/provision/install_sync_gateway.py

```diff
@@ -39,11 +39,11 @@ def __str__(self):
         output += "  skip bucketcreation: {}\n".format(self.skip_bucketcreation)
         return output
 
-    def sync_gateway_base_url_and_package(self, sg_ce=False, sg_platform="centos", sa_platform="centos"):
+    def sync_gateway_base_url_and_package(self, sg_ce=False, sg_platform="centos", sg_installer_type="msi", sa_platform="centos", sa_installer_type="msi"):
         platform_extension = {
             "centos": "rpm",
             "ubuntu": "deb",
-            "windows": "exe"
```

Comment:
we need to have support for exe too, right?

Reply:
Yes we do, look at lines 64-65